packages/devtools_app/lib/src/memory/flutter/memory_graph_model.dart

```diff
@@ -62,8 +68,19 @@ Map<LibraryClass, Predefined> predefinedClasses = {
   predefinedString: const Predefined('String', true),
   predefinedList: const Predefined('List', false),
   predefinedMap: const Predefined('Map', false),
+  predefinedHashMap: const Predefined('HashMap', false),
 };
 
+bool isMap(HeapGraphClassLive live) =>
```

Comment:
can we check whether the class implements the Map interface or does that not make sense due to whether we have sufficient information about keys and values?

Reply:
Added TODO to investigate that to make sure we handle properly.  I'm not sure.